### DISCUSSION TITLE
Discord Server ?

### DISCUSSION THREAD

**Discussion by Snowad14** [Woodi-dev/Among-Us-Sheriff-Mod#9] [Q&A]
Sorry to make a second discussion, but I would like to know if you have / you would like to create a discord where you can put your progress on your mods or just be able to talk to you easily

**Comment by Woodi-dev** [comment]
Hi, feel free to create discussions. I will not create a discord for that. You can find my progress of modding here on github. For further discussions you can contact me via mail

**Comment by blacklambCZ** [comment]
https://discord.gg/S9qPqwHTF4 
I created a discord server for this mod and other mods. You can join here. (Note that if Woodi-dev wants to remove the discord server. I will remove it.)
